### ISSUE TITLE
The link to `http://fictrac.rjdmoore.net/` is broken

### ISSUE THREAD

**Issue by h-mayorquin** [rjdmoore/fictrac#30]
As in the title.

![image](https://github.com/rjdmoore/fictrac/assets/6429509/be1cb550-7c5f-40b2-a6b7-dc15d5735598)

Just in case you wanted to know

**Comment by rjdmoore** [comment]
Yeah, I have changed nameserver for the website - it should be back up
again soon.

cheers,
Richard
